File: static/codes/selection-sort.py
# Traverse through all array elements 
def do_it(A):
    for i in range(len(A)):
        # Find the minimum element in remaining  
        # undone array 
        min_idx = i 
        for j in range(i+1, len(A)): 
            if A[min_idx] > A[j]: 
                mi n_idx = j 
              
        # Swap the found minimum element with  
        # the first element         
        A[i], A[min_idx] = A[min_idx], A[i]

File: static/codes/selection-sort.cpp
void do_it(int arr[], int n) 
{ 
    int i, j, min_idx; 
  
    // One by one move boundary of undone subarray 
    for (i = 0; i < n-1; i++) 
    { 
        // Find the minimum element in undone array 
        min_idx = i; 
        for (j = i+1; j < n; j++) 
          if (arr[j] < arr[min_idx]) 
            min_idx = j; 
  
        // Swap the found minimum element with the first element 
        swap(&arr[min_idx], &arr[i]); 
    } 
}
